src/webgpu/api/validation/render_pass/storeOp.spec.ts

```diff
@@ -0,0 +1,16 @@
+export const description = `API Validation Tests for RenderPass StoreOp.
+
+  Test Coverage Needed:
+
+  - Test that when depthReadOnly is true, depthStoreOp must be "store".
+
+  - Test that when stencilReadOnly is true, stencilStoreOp must be "store".
+
+  - Test that when depthStoreOp or depthReadOnly are set, the attachment must have a depth aspect.
+
+  - Test that when stencilStoreOp or stencilReadOnly are set, the attachment must have a stencil aspect.`;

```

Comment:
depthStoreOp and stencilStoreOp are actually required, which I guess means they're ignored if the format doesn't have the necessary aspect.

Similarly, depthReadOnly and stencilReadOnly have defaults, so in a sense they're always "set". If the store ops are ignored I'd ignore these too for consistency.

cc @kvark

Reply:
Removed these.